src/helpers.coffee

```diff
@@ -136,11 +136,11 @@ buildTokenDataDictionary = (parserState) ->
 # This returns a function which takes an object as a parameter, and if that
 # object is an AST node, updates that object's locationData.
 # The object is returned either way.
-exports.addDataToNode = (parserState, first, last) ->
+exports.addDataToNode = (parserState, first, last, {forceUpdateLocation} = {}) ->
```

Comment:
Let’s not mix styles, of arguments and then an options object:

```coffee
exports.addDataToNode = (parserState, first, last, forceUpdateLocation = yes) ->
```

Any reason not to make it default to true? Then you don’t need it in `getAddDataToNodeFunctionString`.